.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,40 +1,6 @@
-##### SUMMARY
-<!--- Describe the change below, including rationale and design decisions -->
+<!--- Provide a link to the ticket or document which prompted this change -->
```

Comment:
Nit: I think if summary is not the first line in this template, people are like to put their description above it. I am neutral as to whether we even need that heading, but if we have it I think it should be first, and the comments/instructions about what to put there should be beneath it

Reply:
:+1: I just removed it.
(was copying from what commcare-hq does, but I agree that the heading doesn't really add anything)